lib/ProMotion/form/form.rb

```diff
@@ -71,6 +71,12 @@ def input_data(input)
       data[:title] ||= input[:label] || input[:name ].to_s
       data[:cell ] ||= input[:cell_class] if input[:cell_class]
 
+      # valid boolean checks
+      if input.key?(:value) && input[:type] == :boolean
```

Comment:
I was under the impression that `input.key?(:value)` merely checked if the key existed, and not actually look at the value. Is this not the case? PS: not a Ruby pro so please ignore my ignorance :wink: